### DISCUSSION TITLE
[Copilot labs] Command palette brush without selecting custom

### DISCUSSION THREAD

**Discussion by asportnoy** [community/community#47682] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

When I use "Use brush on selected task"  in the command palette, I should be able to type text and have it be used as a custom brush. Currently, I have to select "custom" and *then* I can type my custom text.

**Comment by roberttalling** [comment]
Agree, please add a way to save a custom brush prompt, or save prompt history for easy reuse
